doozerlib/distgit.py

```diff
@@ -1514,8 +1517,14 @@ def update_distgit_dir(self, version, release, prev_release=None):
                     release = release[:-3]  # strip .p?
                     release += pval
 
+                if self.runtime.assembly:
+                    release += f'.assembly.{self.runtime.assembly}'
```

Comment:
I realized this breaks the assumption that `.p[0|1]` is always at the end of the `release` field.
We need to find & replace all `endswith(".p1")` `endswith(".p0")` calls in Doozer code with `find()` or `in` statement.

Reply:
I think I caught them all - everything should be using `util.isolate_pflag_in_release` now.